bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/UL/thq_4f_ckm_LO_ctcvcp/thq_4f_ckm_LO_ctcvcp_run_card.dat

```diff
@@ -0,0 +1,256 @@
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	=  run_tag  ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  False	=  gridpack   !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  1000	=  nevents  ! Number of unweighted events requested 
+  0	=  iseed    ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+  1	=  lpp1     ! beam 1 type 
+  1	=  lpp2     ! beam 2 type
+  6500.0	=  ebeam1   ! beam 1 total energy in GeV
+  6500.0	=  ebeam2   ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+  0.0	=  polbeam1  ! beam polarization for beam 1
+  0.0	=  polbeam2  ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  'lhapdf'    = pdlabel     ! PDF set
+  $DEFAULT_PDF_SETS = lhaid
+  $DEFAULT_PDF_MEMBERS = reweight_PDF     ! if pdlabel=lhapdf, this is the lhapdf number
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+  False	=  fixed_ren_scale   ! if .true. use fixed ren scale
+  False	=  fixed_fac_scale   ! if .true. use fixed fac scale
+  91.188	=  scale             ! fixed ren scale
+  91.188	=  dsqrt_q2fact1     ! fixed fact scale for pdf1
+  91.188	=  dsqrt_q2fact2     ! fixed fact scale for pdf2
+  -1	=  dynamical_scale_choice  ! Select one of the preselect dynamical choice
+  0.33 =  scalefact         ! scale factor for event-by-event scales
```

Comment:
I am not very knowledgeable with this setting.
Is 0.33 set with some reason?

Reply:
We are following the same configuration used in the full RunII card, the purpose of the analyzer is to have exactly the same configuration for the UL cards.

Reply:
@agrohsje Are you aware of this?

Reply:
But do we not need to understand what this does?

Reply:
resolved